include/TPP/Passes.td

```diff
@@ -207,7 +207,7 @@ def TileConsumerAndFuseProducers : Pass<"tile-consumer-and-fuse-producers",
            "Get producers till maxDepth">,
     Option<"startFromLastFusableConsumer", "start-from-last-consumer", "bool",
            "true", "Fuse from the last fusable consumer of the current target">,
-    Option<"useForAll", "use-for-all", "bool", "false", "Use parallel forAll">
+    Option<"useForAll", "use-for-all", "bool", "true", "Use parallel forAll">
```

Comment:
And this has no impact if we don't use threads/OMP, right? IE, it lowers to simple loops.

Reply:
At the end of the pipeline you have `scf.parallel`. It has no impact if we use OMP.